docs/src/rules/no-extra-parens.md

```diff
@@ -130,6 +131,18 @@ for (;(a = b););
 
 :::
 
+### ternaryOperandBinaryExpressions
+
+Examples of **correct** code for this rule with the `"all"` and `{ "ternaryOperandBinaryExpressions": false }` options:
+
+```js
+/* eslint no-extra-parens: ["error", "all", { "ternaryOperandBinaryExpressions": false }] */
+
+(a && b) ? foo : bar
+
+(a - b > a) ? foo : bar
+```
```

Comment:
Both examples are for the `test` node, can you also add a few examples for `consequent` and `alternate`?